frontend/packages/kubevirt-plugin/src/components/vms/vm-console/connectors/VncConsoleConnector.tsx

```diff
@@ -18,6 +21,10 @@ const VncConsoleConnector: React.FC<VncConsoleConnectorProps> = ({ vmi, setConso
       port={window.location.port || (isConnectionEncrypted() ? '443' : '80')}
       // Example: ws://localhost:9000/api/kubernetes/apis/subresources.kubevirt.io/v1/namespaces/kube-system/virtualmachineinstances/vm-cirros1/vnc
       path={`${getVMISubresourcePath()}/${getVMIApiPath(vmi)}/vnc`}
+      textConnect={t('kubevirt-plugin~Connect')}
+      textConnecting={t('kubevirt-plugin~Connecting')}
+      textDisconnect={t('kubevirt-plugin~Disconnect')}
+      textDisconnected={t('kubevirt-plugin~Click Connect to open the VNC console')}
```

Comment:
I think we should also set `textCtrlAltDel`.

Reply:
@pcbailey And what do you think about "VNC console", "Serial console" etc? I've found that we can make them translatable, too, but not sure if we want. Also I've found this:
![vm_down_console](https://user-images.githubusercontent.com/13417815/139949284-0cc55679-c04e-4271-bf43-e21174132c89.png)

Reply:
@hstastna If we can make them translatable, then I think we should. The same goes for that message. I would typically say put it in a separate PR, but since we have to open new bugs for every PR we should probably go ahead and do it in this one.